source/jormungandr/tests/journey_common_tests.py

```diff
@@ -1482,18 +1482,18 @@ def test_min_nb_transfers(self):
 
 @dataset({"min_nb_journeys_test": {}})
 class JourneysMinNbJourneys():
-    """
+    '''
```

Comment:
PEP 257 : For consistency, always use """triple double quotes"""
I won't put a comment on every changes but please revert this